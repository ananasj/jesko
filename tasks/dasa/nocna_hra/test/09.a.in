1243 12
k
fuivjpzcwqby
yc
von
c
gdtpvzzyfgtp
feibw
dwahrtb
mdwfssgt
bldg
lzl
n
pvajwva
zntrwpjl
ozoy
mvy
frf
ccc
von
von
xuyoe
d
ry
mn
von
lhes
ultyn
jrnp
sczkdqrp
fekmjcvrzhkz
wfwjvb
ddzgwrybqd
srfn
utdqybyxm
ridwiks
wtpew
cvdi
okxriuedu
xvvjqw
s
osn
qbhprrmibi
enpdjlnzjin
fsoukdknkb
pn
von
kdpcha
bag
von
vwqpi
wgdalspu
exiigi
l
zcpw
von
ic
ktjwoysir
bholgswefx
von
kdivgxbqsk
lgocf
omr
qbmnl
wo
uqsmjfbpqjgu
aequzwk
znvijjc
svtbcwtsgzoy
dtlervgcutxf
h
von
ekdghw
xp
gsvrw
eqiap
zwoowv
von
ix
wt
zdtsb
i
von
pgppcfdaaeb
aijwbwucb
cxckxsumalb
hdtnhx
flei
mkoipnkzk
yfhckal
yvvaeboilxh
xgz
awywwd
o
ee
amfd
q
omuvlgol
k
qbha
haojnwmp
rjtdfqqnecr
tlewt
gllx
tis
lqgpxy
cjdwxyjdv
invtkswfaom
exqdvy
jsxjshmn
vyu
mprodfgeldx
von
dygx
qfgfscdtyw
lxbqehu
von
gl
mxjkq
wjjb
zkimvfnllwh
b
ruiqfshjj
sggxg
snmhax
zlmjhgfpwmjf
lrn
kzrrre
gcf
tcmaktsjib
wb
vyvuqmowrodt
xpzlpkhjtrlj
nxsi
cqds
von
rpfo
o
von
lvvcigcptaj
ygq
e
yoom
chuesrzwzgy
abdczjuc
ztznhlssqbou
von
tgcuyduzizy
adsb
qhhlzxmp
updcr
xuzftzzmb
ecavlhil
a
xpkbjhegb
ufmjisqn
k
utgqvix
hwh
pqhem
vxlnmigwqi
qigzfbzmzg
xtidhx
uisgqaeg
rbexckabw
zciull
ijesdkieq
mhtsgvc
b
cnpxybbilk
von
wntmdmam
udxdz
jabyzbbcjmo
etez
lhvt
w
vyjgbmhc
fwxtt
mv
cmca
qbelzdxfgjm
xnvu
kibf
kwbk
cd
cr
zywhvtvs
hzolfai
von
bxtd
seahozfmgcfd
jevapaa
yksvdxymqemz
ggwknz
lkfn
von
yznlrk
idhprijfosdy
epprueq
von
ag
tuldxssra
qwbqd
von
xnobgsvzs
von
wxzaxruot
lunbauysrnj
bq
txksvjusdoiy
kll
fnexbpswqi
von
oxxjhr
inkxjwiuk
von
azcptzfb
geqeqzvvldl
uslreft
ugwnhdqzvwfn
mytxdevgan
tykyfggdtn
od
rirdikd
yutnl
l
sfnvzcdkr
lzf
npibhgvati
tmhn
ksznfqw
vehasy
gq
lzxmt
i
von
vrga
amaeuayuluc
lpmkdf
qzsjujpxvuxh
dweyja
nngz
hulxtd
ox
rl
rwmvwx
mkmsjk
gpnzsvr
fukrnail
ilch
tsvewom
ofzf
kkedzglmu
gud
x
dxspk
von
u
oxutatzoh
e
hihgillfddqb
vtqksmf
egoans
von
m
von
jucoxupfclau
g
ni
lgunu
sfxdzzrwu
yuetrw
hrdbtojpegmg
fdseuqabtb
wouxjdopgj
zlwnr
wzydvqfxj
jcjfcuiqlrc
von
cfow
ueu
kfwvlteo
qauimlkfunhb
gzakvzlhek
von
jghv
pi
t
ikbmrhl
hteoxotptdvc
mpehtzkr
uvndhhwqa
xtanm
pdi
mknnwgu
von
ifzrd
fxqyxfncse
hbmtqbqyx
p
onifeo
occbfufouosi
jah
h
nnwv
lzac
d
lxtbpejanj
wsvffkd
gqigsmlyv
uf
jwo
xxdpukxeoa
sxfljqjgcua
orecs
prrkcoqs
oiattl
rnauxjiop
bmxtdhxtypj
se
pksj
d
ln
vwnsrsaom
xblrfkfxwzgw
tz
u
qmui
yuugsvr
von
von
czpkwlvrk
gubwgxgpt
pnblelqhbsit
von
zwquwwpzuxx
skwizzvfmlo
wjjdpkbfgxdx
xv
qohypj
v
jnh
k
lqq
oivlgxwwog
prakxi
rqkbauoqkam
hl
inw
afaapzknlcex
rwydw
ftp
zja
von
u
ktpxgtdda
wka
easvxwv
kdzfzdr
obnufodbk
dmjgef
any
von
rubinvzo
vx
hocncmtitykt
hiizcjjre
pddmec
gjw
encqi
von
zgrbrasylan
cujyor
mnqbrgkih
ptljvghiiwb
vywmpfydurf
von
ugteyorh
quhmvwnswj
rhrny
xp
von
von
gaspmsfi
fy
usgilncy
x
usevafqpt
von
joijwheo
bcqaprz
uwppxufmppjs
akvi
von
hnaxpr
tdjs
p
dfxoxynal
bzhio
von
bavwe
x
omltkbtipttc
dkhrwpxxss
ymvft
gfdi
zvhbaegmo
dxfxrdtspqx
qrjuz
y
bgwjsk
phinblitcbsn
gmghxohdg
ctdobgfjps
von
fhez
jc
jueszmc
vneyukhletn
wnzswjutzfp
osbaiyqpxm
xfcmp
ecdb
w
othe
fyh
ghqogvqskt
xhskhqyogt
jesjnnyvwojc
uyvanuhheoz
iza
gltva
whb
bwrywg
pytocyyearog
bgz
huecy
wxzr
pye
d
von
mebwhjy
fvs
vcrv
yrayv
yw
kcahljfmxm
ven
iaii
hoggpcd
h
tv
vmqnhuagrlgz
jcopidrore
zq
xvmnk
nc
lat
wcb
twyzsdyk
i
von
ol
pibciwpu
von
bzpxxqrdoch
kejyrqbgy
hb
vrzyqqyq
txtcohourp
stgx
ac
wswoiwernzqh
osahjnpbgvy
a
ylywajsga
q
wffwmolbqryq
rurrh
jyjqaoaq
okrmyh
von
rcqxpjtgabvr
acwqcpv
hjsiqp
imggbrz
chkpdk
cyqa
jxorp
fo
utevdxykjnpv
yghyufhtt
von
esmugiqmetm
vturbm
von
rcaayt
ugbicj
yxnkb
uqavdrdykibl
von
r
szzestcpjmq
kwyfzp
ctcnbzbsrwms
kqsbbgr
ddcletwdvrg
hkm
a
tdxnfzuygael
pxnstgad
poixod
gevnfronrs
jnsw
ekre
sdiigmqpsne
s
mmuvzotj
von
vdodvulecawr
yupsgo
kcuy
cv
zrfwlscpuy
jhjazrog
j
jodgpffgkete
zhtqq
rcsf
von
vojecml
sz
u
fjmyzcbatfub
von
lp
ugcn
phtjryoc
dpsdix
obwzrqrn
abqpikza
ex
mynwvnikpgl
evya
snhe
pfommy
lwibg
nfcmmzk
von
von
lslbe
uovuohhk
rlcgteuff
ep
jktpmc
xfqen
xwhacagwf
astajckcu
fbblu
x
von
nx
gtvuxwmqy
kao
wnijhshew
lewzkr
ovsgt
dprpupc
j
nunmfrkh
ej
dzutbxis
jqihzeooyec
mlbprm
q
jhl
von
ynqgusllglp
mvcplv
nu
fipylrwy
vbqgib
tloxbbtfvizw
gberudsdi
yvpbsiuf
g
lvvmthszkz
cmkkbti
von
qugw
esnsp
kszleygla
kwquuafnun
nphch
ojfpujncupvr
jvnjcafslu
bkkbxa
phncjitgb
lppwupcmbw
zfqacrngiwu
eeof
ssepmw
xrwdqbtse
kvbgijkozqpq
x
tuxsltvdwqw
bnyfvgqguryk
sgk
gntugq
ht
xwkcsquymly
von
qdshg
a
ssiz
lj
ngphecucaqkt
acgueb
thyhxlslj
yoxakb
nw
wcasidsfzzf
qzjzgcaxb
edmsbnmxrn
von
khpjmyi
tehy
bgo
cyn
njzdjsnshye
igemo
udzijln
cwnnvsy
qxgweq
ut
oxcninavxcz
pqk
s
aajqhwlb
rzlejb
yihta
lk
qsfq
aaid
ubfam
sgqbzryle
jav
x
xsv
pct
axkiomkhkvl
uu
m
von
von
naqzsvtjlvhw
von
peumpqklmet
ofnciddayyl
skbpuqtqfi
tsvosrfyeh
mkklvwgoi
ca
hhcyduttp
prhxdujof
nuvxyzxxu
von
c
xfkkdzelyhfh
niz
wi
fcbn
zu
gnjft
bd
von
bgrj
tsly
o
nrxtzmc
z
czfu
lmzaitn
cwn
von
vc
hnrg
von
aigdu
vwzrjihn
jqfgumqbb
bk
sceowy
von
cdhjrmiccn
bback
ufh
ocgmo
qftcyhmdlz
dpjhbvnyavmk
szhukzcfbc
fzpvjskuzo
ms
evrzuyvgy
bccrjcifn
jdfbsr
szzlavmwe
spzuglyqsl
vwrwokrtessq
etuqqmf
o
evsxepvxnjig
von
xewniphaftiv
a
vdbauwxii
mwrja
lcdliyt
rgmubrvdmud
kooggajtw
chgcdne
tqxkoc
f
slcatcjob
d
fskmoff
pxfqhmmve
hses
yv
qpjadzfiikaf
ofejmelty
hu
eavvjx
ejto
hbnnizzmks
fuvrxbtuyf
von
vjj
k
a
zhulbgm
h
itviymiokt
pmr
avn
ylym
von
uedn
m
von
ypewerp
prtbjt
gilzcrfprtdf
fgmdmds
kaellx
dhjfl
auf
uogaalogdx
zvaksbiyktdv
youe
sxquieaod
cogojjwh
krmzpdv
lnubi
ku
ypsgedpck
wdjxums
dfegploaf
fsy
cdboovl
ioubgdg
von
bu
aejfwjtvnwwc
von
gc
w
zhpju
xv
elboew
udqqswsqu
yqzqztiesek
xxwdtahqdxg
aqqoireih
pmutyrev
ykcfagcieyk
ye
mrmq
mcw
tsitenaet
zfy
yzgfoxs
von
qirtqmnag
gxinljtler
y
vjabeqlxjbj
rqryoan
gpqybywsut
von
kkihvmqu
nhi
vgtevlcwja
vvxu
foergbjqosar
zvc
gtrgkud
yoafcgwjhh
zcobyoyclgie
rttymtpoy
wcedecfgsg
hitpqan
iztvvijtcgqe
knashjyqsi
alwg
iyadilyksqrd
so
zwrt
bf
ovew
bdszqmshrgm
iky
rmhsrxs
y
fgvzinlawcgi
rzcyduk
von
igjefowmvt
gwdeal
pfmsdqonaark
r
ncl
oolt
ge
mmqpc
fgyrraihqxje
urlqlww
u
kvlq
xakoa
t
zxitrtlcsiwo
xpwlfyfezrv
tugcvfk
gabykxm
meyxlwesxwn
von
vqkrvvi
kxbxunmwbsua
fxsbmr
pbdiy
sdcpezmtnkv
vnaams
mmtb
loydg
yjjmdw
e
tdwf
eioa
ywtzbcciamrm
kpw
p
wnkdvad
bezafbfo
hpkxzbtn
von
yhsikqinynqd
in
qtihffjgb
von
von
tuydeontmd
rfah
zaihfqqnr
csexqjdezzs
iv
cxmcxvlelbt
von
gj
von
vlfukyyv
nuakge
pnqsiujl
vkwsvc
cmemhr
dolmuc
ujfgxkbuuxm
yzdmgr
zdboqpitrupa
zl
von
w
von
fbdrhwnhws
kzu
ovt
sdrmlptno
tsaqhhmbry
ur
oecjzgcnt
jhhabcatj
ldfz
silhoosnwu
nlyusaxu
g
tqes
kxttlhlbff
uovpj
fyfll
clwrsjolcbu
zxgws
jfox
cwyjdcubtmkj
lu
we
wul
zfmmjjns
mp
cwdr
yz
wpbqais
eozpbrdtgveh
mqmlolhsjyv
gs
ypuotorm
s
von
p
von
rtj
ihfjnz
xbhlwvfi
cudxlg
ybzkghr
hkeuhibveza
rrn
yfyodbonh
sqpewjn
ilmicccvraoq
g
uz
gqkvwgh
usagcckexc
uozazo
m
czcyhjkwfcw
yxn
ultinwjbri
o
stodflihjv
von
cisy
h
qkdac
wshrilwvvffr
kxzeyusfde
jatuppoyhz
von
von
von
von
yeui
obqjivnimz
tzuk
ljhhqslqwh
do
xnsnvemflheh
von
ensofmbviivl
qilgaybg
rvnzrr
eg
sogczdnopg
mza
o
lmac
c
pvmeeofhety
aws
cqmd
von
gfuip
n
tujaacbmv
pex
j
jchpz
qpvttopcrph
exnvbloyukyg
xo
von
mbz
pkkgre
cxqfkedfreng
von
ipavq
iqnszhzl
kvgvbmau
adzw
yjhao
dlrkkcpnp
kpi
abmda
qjcxjqoc
vgmavap
zzell
xjqkdgu
f
jsyhynhwn
lxerkrxypi
luvtcdrz
xvjkefzclm
djjc
von
xbau
we
svvqfhwk
xvyrbk
vcsvfv
o
von
lfwixdptqy
wtgxficamxvu
agalj
ixploib
rlwzibkkbwhz
von
tingtcybznno
von
ahowjy
idrscktpdddj
jy
nxveawn
von
o
yzhqdrmswpxh
qnmoaj
lhsdc
ye
nk
wwcnujcbz
pp
wli
kehkkaj
von
dijf
qygppvge
cinsyz
lj
hujmfuu
kbi
g
dpzoerm
uqaemjavvfq
scj
bkmyq
egqs
ochspqtmna
jwhsblunzu
tjkjcy
czjrreg
kmpjtik
xhydlrmxcov
xuqrmwzrcj
syccgic
von
ab
dkuyrstjlfhk
vddodh
rwazdbaz
jtccn
acumcfhhlvks
kmwlqxmp
aswefup
ltbqbib
uo
botrnfilzid
von
qdsoom
nyi
cclkdadwnlea
g
rwhjlvx
maja
chzhhf
qjzcxflzy
j
hx
zsvvhzeh
mljeukirpt
ba
diwm
ebtlaxu
i
von
pgxixqbntbo
ypvd
egmffh
rkngzvdist
nmxgvvvq
pfremzncqe
kmapwv
cyucgpxb
von
xggdsfqvv
bjwdztykadi
omerufgtlmzf
cqomtajxbe
gctdjjrqan
giu
pzsgpezp
qtwbayw
von
wy
vqrfnz
yoknt
ik
gzghvlr
rlsmblto
ti
yxpdha
von
zrrveiynblf
riddywo
mbrritrubr
osfmfhxn
gfojfmxjoj
auauodleyzwe
nmqj
qqixcfg
sbtn
dh
fb
ojacbkcacuka
gribsc
rrdszktgwkr
von
iqvlkh
wvcfwujkqaev
golcyd
yldvypclyai
fxqrpr
larzmvxp
pipkgemtd
fuj
audwfpwyo
mzqyqbnahr
wdfc
ijqcnmj
jsvcex
nwpqfjeifjp
fsswwhkil
dtwkqrofpg
pchul
ueyocwxme
wndszvloacu
jahfngljm
oi
ogk
pevkr
vdlwondu
hkxtjndg
occksigecp
vctlpzfs
ek
zlvpekuiumrc
ukqpomaeliyv
hhrhu
von
von
shvgayzt
ljxx
ktbisfpzxytm
nrzia
zucihnh
lruobn
mtrihqau
ekcpjwtt
cmgngaeh
acvukemmywn
p
npouwr
gpgkwwmzaht
fylensmeni
wnanw
lngkjtl
exolvbp
eojo
kwoatu
hcqsx
ahaqvxtkjp
yggnrdbt
pchtv
yftiijdhcprr
roxff
ayi
qlxse
xghijrmyig
x
ecxndxxuanhz
xhmdqw
jlrprsomptrp
ofqovaocnnl
eyfn
hyk
yqiphsly
mueczrs
jq
edh
rmtredjovum
atafsuxgrgx
dtdqxwcqqiv
von
tiosjhuqzqn
vfvrk
von
lvedl
mjq
faemuuolhw
dqwnhvp
tyefyqqraky
mxl
uafzdvx
shx
xoi
yqhkqiddhqhr
wlnti
apxls
avgamolc
von
fgpysz
mpo
von
kkacibeeke
r
hefpvdiuz
o
ernqtrzxvd
vhtthcxmsurc
qxgsckm
hiaeff
cub
gvpypgcg
