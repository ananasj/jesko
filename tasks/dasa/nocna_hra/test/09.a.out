malo deti
pvajwva
malo deti
malo deti
jrnp
s
zcpw
malo deti
svtbcwtsgzoy
malo deti
malo deti
pgppcfdaaeb
amfd
invtkswfaom
b
nxsi
adsb
vxlnmigwqi
jabyzbbcjmo
cd
epprueq
malo deti
malo deti
inkxjwiuk
lzf
dweyja
kkedzglmu
malo deti
g
ueu
pdi
d
bmxtdhxtypj
gubwgxgpt
prakxi
kdzfzdr
malo deti
cujyor
malo deti
malo deti
usevafqpt
malo deti
malo deti
omltkbtipttc
fhez
xhskhqyogt
d
vmqnhuagrlgz
kejyrqbgy
rurrh
yghyufhtt
ddcletwdvrg
vdodvulecawr
u
pfommy
malo deti
x
jqihzeooyec
g
lppwupcmbw
xwkcsquymly
edmsbnmxrn
pqk
xsv
hhcyduttp
malo deti
tsly
malo deti
bk
bccrjcifn
lcdliyt
qpjadzfiikaf
itviymiokt
auf
fsy
xxwdtahqdxg
y
zcobyoyclgie
bdszqmshrgm
ge
meyxlwesxwn
eioa
zaihfqqnr
malo deti
yzdmgr
malo deti
ldfz
lu
s
g
h
malo deti
malo deti
malo deti
qilgaybg
n
malo deti
iqnszhzl
jsyhynhwn
wtgxficamxvu
malo deti
malo deti
lhsdc
kbi
xhydlrmxcov
uo
zsvvhzeh
kmapwv
yoknt
nmqj
fxqrpr
pchul
zlvpekuiumrc
acvukemmywn
yggnrdbt
eyfn
